src/index.ts

```diff
@@ -119,6 +146,23 @@ function parseLineHeight(value: string) {
 	const parsed = parseFloat(value);
 	if (parsed.toString() === value) {
 		return parsed;
+	} else {
+		const match = /^(\+|-)?(\.)?/.exec(value) as RegExpMatchArray;
+		let val: string = value;
+		if (match[1] === '+') {
+			val = val.substring(1);
```

Comment:
This shouldn't be needed once we deconstruct the match into `[sign, value, percent]`.